app/components/forms/events/view/videoroom-form.hbs

```diff
@@ -31,8 +31,14 @@
 <br>
 <form class="ui form {{if this.isLoading 'loading'}}">
   <div class="field">
-    <label class="required">{{t 'Room'}}</label>
-    <input value={{this.roomName}} readonly>
+    {{#if this.microRoom}}
+      <label class="required">{{t 'Room'}}</label>
+      <input value={{this.microRoom}} readonly>
+    {{else}}
+      <label class="required">{{t 'Room'}}</label>
+      <Input
+        @value={{this.data.stream.name}} />
+    {{/if}}
```

Comment:
It should always show stream name now, not room name

Reply:
then if microlocation rooms are renamed from wizard they dont show up renamed here

Reply:
OK so now for my events, I renamed them earlier using wizard, so 2 names appear, table shows a different one and edit form shows a different one. Is it ok?

Reply:
Yes